### PR TITLE
{vis}[system/system] OptiX v7.2.0

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/o/OptiX/OptiX-7.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# https://developer.nvidia.com/designworks/optix/downloads/legacy
+sources = ['NVIDIA-OptiX-SDK-%(version)s-linux64-x86_64.sh']
+checksums = ['6355a03b2819a3720276c0ca9358641635caf99b8e760b43d9a6e04e1ec78b85']
+
+install_cmd = "./" + sources[0] + " --skip-license --prefix=%(installdir)s"
+
+sanity_check_paths = {
+    'files': ["include/optix.h", "include/optix_device.h"],
+#    'files': ["include/optix.h", "include/optix_device.h", "lib64/liboptix.%s" % SHLIB_EXT,
+#              "lib64/liboptixu.%s" % SHLIB_EXT],
```
Comment (Micket) [inline easybuild/easyconfigs/o/OptiX/OptiX-7.2.0.eb:27]: does these not exist anymore? surely there is some library we can check for at least.

**Review by Micket — DISMISSED** [review]

File: easybuild/easyconfigs/o/OptiX/OptiX-7.2.0.eb (hunk 1 above)
Comment (robert-mijakovic) [reply easybuild/easyconfigs/o/OptiX/OptiX-7.2.0.eb:27]: The installer script of the previous release of OptiX, 6.5.0, created a lib64 directory, while the newest one, 7.2.0, doesn't. Searching inside of the installation directory structure doesn't show any libraries, shared or static.

Comment (robert-mijakovic) [reply easybuild/easyconfigs/o/OptiX/OptiX-7.2.0.eb:27]: That is the reason why I commented it out in the first place. I tried to use it with VMD but, as expected, it doesn't work.

Comment (Micket) [reply easybuild/easyconfigs/o/OptiX/OptiX-7.2.0.eb:27]: Seems like optix 7 is is a complete redesign and is baked into the nvidia *drivers* (`/usr/lib64/libnvoptix.so.460.32.03`). Not sure what the "SDK" does anymore.

Comment (Micket) [reply easybuild/easyconfigs/o/OptiX/OptiX-7.2.0.eb:27]: https://forums.developer.nvidia.com/t/optix-6-5-cannot-recognize-device/157877/5  alright so, they confirmed that Optix is header only starting with optix 7. So. Just remove those commented lines and this PR should be good to go.

Comment (robert-mijakovic) [reply easybuild/easyconfigs/o/OptiX/OptiX-7.2.0.eb]: @Micket Done.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3101.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/a1edfe315c152eb850c86449277a5397 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**